service-api/app/test/AppTest/Service/Lpa/CheckLpaCleansedTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace AppTest\Service\Lpa;
+
+use App\DataAccess\Repository\Response\Lpa;
+use App\Exception\BadRequestException;
+use App\Service\Lpa\CheckLpaCleansed;
+use App\Service\Lpa\LpaService;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use DateTime;
+
+/**
+ * Class CheckLpaCleansedTest
+ *
+ * @package AppTest\Service\Lpa
+ * @coversDefaultClass \App\Service\Lpa\CheckLpaCleansed
+ */
+class CheckLpaCleansedTest extends TestCase
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $loggerProphecy;
+
+    /**
+     * @var LpaService
+     */
+    private $lpaServiceProphecy;
+
+    public function setUp()
+    {
+        $this->loggerProphecy = $this->prophesize(LoggerInterface::class);
+        $this->lpaServiceProphecy = $this->prophesize(LpaService::class);
+    }
+
+    public function checkLpaCleansed(): CheckLpaCleansed
+    {
+        return new CheckLpaCleansed(
+            $this->loggerProphecy->reveal(),
+            $this->lpaServiceProphecy->reveal(),
+        );
+    }
+
+    /** @test */
+    public function older_lpa_add_confirmation_throws_an_exception_if_lpa_not_cleansed_and_registered_before_sep2019()
+    {
+        $userId = '1234';
+        $lpa = new Lpa(
+            [
+                'registrationDate' => '2018-05-26',
+                'lpaIsCleansed'     => false,
+            ],
+            new DateTime()
+        );
+
+        $actorDetailsMatch = [
+            'lpa-id'            => '700000000001'
+        ];
```

Comment:
```suggestion
        $lpa = new Lpa(
            [
                'registrationDate' => '2018-05-26',
                'lpaIsCleansed'    => false,
            ],
            new DateTime()
        );

        $actorDetailsMatch = [
            'lpa-id' => '700000000001'
        ];
```

Need to be more consistent about spacing around lines. The spacing is always dictated by the longest line in any given block.